Documentation/cheatsheet.rst

```diff
@@ -0,0 +1,137 @@
+#################
+Cilium Cheatsheet
+#################
+
+Below is a short list of examples of the various commands Cilium has to offer.
+If a command you use is missing please don’t hesitate to add it to one of the
+groups or create a new one.
+
+Basics
+======
+
+::
+
+        # Check the status of the agent
+
+        cilium status
+
+::
+
+        # Get the current agent configuration
+
+        cilium config
+
+Policy management
+=================
+
+::
+
+	# Importing a policy
+
+	cilium policy import <my-policy.json>
+
+::
+
+	# Get list of all imported policy rules
+
+	cilium policy get
+
+::
+
+	# Remove all policy
+
+	cilium policy delete --all
+
+Monitoring
+==========
+
+::
+
+	# Monitor cilium datapath notifications
+
+	cilium monitor 
+
+::
+
+	# Verbose output
+
+	cilium monitor -v
+
+::
+
+        # Show notifications only for dropped packet events
+
+        cilium monitor --related-to=<id>
+
+::
+
+	# Filter only the dropped packets
+
+	cilium monitor --type drop
+
+::
+
+	# Don't dissect packet payload, display payload in hex format
+
+	cilium monitor -v --hex
+
+Endpoints
+=========
+
+::
+
+	# Get list of all local endpoints
+
+	cilium endpoint list
+
+::
+
+        # Get detailed view of endpoint properties and state
+
+        cilium endpoint get <id>
+
+::
+
+        # Show recent endpoint specific log entries
+
+        cilium endpoint log <id>
+
+::
+
+	# Enable debugging output for the endpoint
+
+	cilium endpoint config <my-ep> Debug=true
+
+Tracing
+=======
+
+::
+
+	# Trace between two applications on port 80 for connectivity issues
+
+	cilium policy trace -s <from> -d <to> --dport 80
+
+Loadbalancing
+=============
+
+
+::
+
+        # Get list of loadbalancer services
+
+        cilium service list
+
+BPF
+===
+
+::
+
+        # List node tunneling mapping information
+
+        cilium bpf tunnel list
+
+::
+
+        # Checking logs for verifier issue
```

Comment:
It might be worth having a "getting logs" section or subsection (under Basic). It would be `journalctl -u cilium` for host-level installs, and `kubectl logs -n kube-system ds/kubernetes` for a k8s install.

Reply:
Can we add that later? Can create an issue to track it if you want.

Reply:
Yeah, it was just a suggestion, nothing is blocking this PR.

Reply:
Created a ticket for it here https://github.com/cilium/cilium/issues/2182. 

I think the only remaining thing is to add the one kuberenetes example Thomas wanted for tracing. Need to setup a cluster before I can test that. Would be great if you could review the monitor README again. 

Thanks.